### ISSUE TITLE
Multiple tabs on record page. 

### ISSUE THREAD

**Issue by nlamb2** [burkybang/SuiteAdvanced-Feedback#36]
**Describe the bug**
It appears that when there are several tabs on a page, you can scroll over right to view the tabs, but cannot scroll back left. 

Answer: 

___

**How to reproduce**
Navigate to record with several custom tabs
click on tab in far right
observe that it scrolls the tabs for you
Click left most two tabs
Observe that the far most left tabs do not scroll back

Answer: 

___

**Expected behavior**
It appears to work some of the time, but i would expect the scroll left feature to work the same as the scroll right. 

Answer: 

___

**Screenshots**
I have attached a record to try and help demonstrate what I am seeing. 

https://user-images.githubusercontent.com/56122264/179998246-10793ee2-b12a-47fa-823f-7f58893e403b.mp4

**side bar
I love what you're doing. This plugin is a game changer and greatly improves the development experience.

**Comment by burkybang** [comment]
I tried really hard, but I'm not able to reproduce the issue. Are you able to consistently reproduce it?

**Comment by nlamb2** [comment]
Yeah it does it every time for me. I’ll disable all other plugins I have and retest. Is there any other info I can grab for you to help?

**Comment by burkybang** [comment]
Can you open your browser console and then reproduce the issue? I'd like to know if any JavaScript errors occur.

**Comment by nlamb2** [comment]
Alright, I disabled all plugins except SuiteAdvanced for NetSuite and reloaded without cache. Attached is a recording of the 'possible improvements' 

https://user-images.githubusercontent.com/56122264/180104551-b32dbad4-10d1-440f-85e1-1d799d99ed7a.mp4

**Comment by burkybang** [comment]
Okay, so no JavaScript errors. Are you able to use Shift + Mouse Wheel to scroll horizontally on the subtab bar?

**Comment by nlamb2** [comment]
Yes, that works perfectly. Now that I know I can do that, I'm not really worried about the auto scrolling xD

**Comment by burkybang** [comment]
Okay 🤷‍♂️

I guess since there's a good workaround and since I'm not able to reproduce it, I'll close this issue for now.

Thank you for being so helpful!
